### ISSUE TITLE
Accessibility: all SplitButton example code needs title in iconProps

### ISSUE THREAD

**Issue by kenotron** [OfficeDev/office-ui-fabric-react#10494]
### Environment Information

- **Package version(s)**: 7.x

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Splitting out #6356 to smaller bugs to be fixed. This one concerns the example area and needs a title to be placed in the iconProps.

### Repro

1. Open 'https://developer.microsoft.com/en-us/fabric#/components/button' in Edge.   
2. Navigate to the 'Split button with icon and custom styles' section under 'Implementation examples'.    
3. Openkeros for web tool.   
4. Observe the issue.

### Actual

Name of the 'Custom split button' is not defined. Narrator speaks 'Split button' only which is not conveying the purpose of the control.

### Expected

Name of the 'Custom split button' should be defined.
